### DISCUSSION TITLE
How do I change the px to rem ratio?

### DISCUSSION THREAD

**Discussion by yahirgnz** [master-co/css#48] [Question]
Hey, my app has a different px:rem ratio applied, like:
```css
html {
    font-size: 62.5%;
}
```
So when I'm using Master applying a class like `f:15` results in something completely different from what I wanted, is there any option to configure how the px -> rem conversion works so I can stop writing the `"rem"` suffix for numeric values? 🤔

**Comment by 1aron** [comment]
@gammx We should provide an option to customize the conversion factor for this [line](https://github.com/master-co/css/blob/main/src/utils/parse-value-unit.ts#L17), right?

**Reply by yahirgnz** [reply]
@1Aron Yup, it would be something like this:

```js
  current / (unit === REM ? 16 : 1) // f:20 would give 1.25rem
  current / (unit === REM ? 10 : 1) // f:20 would give 2rem <- This is what I'm expecting
```

I just thought @master/css should be able to handle different rem:px scales since many sites use tricks like this to use REM values with a closer ratio to pixels :)

**Reply by 1aron** [reply]
Nice feature request! The task has been added and we'll support the option asap. Thx. 😊

**Reply by itsezc** [reply]
This would be amazing!

**Comment by 1aron** [comment]
🥳 `@master/css@1.36.0` is now available for the feature.

**Reply by yahirgnz** [reply]
Nice ✨ I was looking for something like this to quickly style some parts of my app, I almost went for Tailwind but found you guys on Twitter, Master CSS has a lot of potential, keep it up:)
